test/models/apple/episode_test.rb

```diff
@@ -85,14 +85,4 @@
       assert_match(/noImp=1/, apple_episode.enclosure_url)
     end
   end
-
-  describe ".add_no_imp_param" do
-    it "should add a noImp query param" do
-      assert_equal "http://example.com?noImp=1", Apple::Episode.add_no_imp_param("http://example.com")
-    end
-
-    it "should preserve existing query params" do
-      assert_equal "http://example.com?foo=bar&noImp=1", Apple::Episode.add_no_imp_param("http://example.com?foo=bar")
-    end
-  end
```

Comment:
Moved these over to the enclosure url builder to illustrate the coupling of apple api related code and enclosure nuance.